pyphare/pyphare/pharesee/plotting.py

```diff
@@ -60,7 +61,21 @@ def dist_plot(particles, **kwargs):
               bins=kwargs.get("bins", bins),
               weights=particles.weights)
 
-    sig = kwargs.get("sigma", (0,0))
+    if "gaussian_filter_sigma" in kwargs and "median_filter_size" not in kwargs: 
+        from scipy.ndimage import gaussian_filter
+        sig = kwargs.get("gaussian_filter_sigma", (0,0))
+        image = gaussian_filter(h.T, sigma=sig)
+    elif "median_filter_size" in kwargs and "gaussian_filter_sigma" not in kwargs: 
+        from scipy.ndimage import median_filter
+        siz = kwargs.get("median_filter_size", (0,0))
+        image = median_filter(h.T, size=siz)
+    elif "gaussian_filter_sigma" not in kwargs and "median_filter_size" not in kwargs:
+        image = h.T
+    else:
+        raise ValueError("gaussian and median filters can not be called at the same time")
```

Comment:
not super important but : I would tend to think it is not this function's business to decide whether its caller is right or wrong to want to use both filters.
I would have simply done if gaussian apply gaussian, followed by if median apply median, and delegate the responsibility outside this function, which only aim is to plot what's askked.